components/sup/src/main.rs

```diff
@@ -1480,6 +1493,7 @@ organization = "MY_ORG_FROM_SECOND_CONFG"
                                        http_listen:
                                            HttpListenAddr::from_str("3.3.3.3:3333").unwrap(),
                                        http_disable:          false,
+                                       http_startup_timeout:  HttpStartupTimeout::configured_value(),
```

Comment:
In all these tests, is it the intention that the HTTP startup timeout, _as a fundamental part of the test itself_, be sensitive to the value of its environment variable?

If that is _not_ the case, and the value of the environment variable is not important to the test, these `configured_value()` calls should be replaced with `default()`, as that always returns a stable value. 

`configured_value()` should only be used in tests where you're actually looking at how different values of the environment variable affect the behavior you're testing.

Reply:
applied in ba5c5ffa74d08bdc53af7733f335fcc78bd893e5 @christophermaier